JoinColony/colonyCDapp
# Cleanup and refactor Manage Permissions

## Description

Cleans up imports and refactors the `managePermissions` saga, adds and refactors Schema Models, wires up the Actions list and page.

I've left comments where relevant, especially in the Schema file, as the changes to the models were less than obvious.

### Screenshots

![Screenshot from 2023-04-24 07-52-24](https://user-images.githubusercontent.com/1193222/233906015-589c540b-dd03-4ef1-b7c3-ff20d0f3d84e.png)
![Screenshot from 2023-04-24 07-52-30](https://user-images.githubusercontent.com/1193222/233906020-bd09f1f2-995c-411d-9b01-94503b76a3c0.png)

### Testing

- Pull everything
- Start the dev enviroment
- create your user 
- create your colony _(I've noticed that the first ever colony created on the chain acts weird, and the ingestor doesn't always pick up it's events, so I'd recomment actually creating a second one, at which point it will work flawlessly)_
- Press `Test Permissions #1`
- Press `Test Permissions #2`
- Refresh the page 
- You should now see / click various actions entries for permissions _(Note that the entry that gives permissions to the extension, actually says that it gives permissions to the colony. This is visually "wrong", the data is all there, but the `FriendlyName` component doesn't know how to display it -- yet)_

Resolves #234
Resolves #235 
Resolves #236
Resolves #237
